src/main/java/com/dropbox/core/DbxWebAuth.java

```diff
@@ -296,6 +296,21 @@ public DbxAuthFinish finishFromCode(String code) throws DbxException {
     }
 
     /**
+     * Call this after the user has visited the authorizaton URL and acquired an authorization
+     * code. This method allows passing both the redirect URI and state.
+     *
+     * @param code The authorization code shown to the user when they clicked "Allow" on the
+     *    authorization, page on the Dropbox website, never {@code null}.
+     * @param redirectUri Redirect URI used when generating the authorization code.
```

Comment:
What's `redirectUri` for?  If the authorization code was shown to the user, there was no `redirectUri`, right?